### ISSUE TITLE
Add "Show description" action in context menu

### ISSUE THREAD

**Issue by optizone** [radareorg/cutter#1599]
**Is your feature request related to a problem? Please describe.**
Now there are only two ways to check what opcode does: 
- google this
- enable `asm.describe`

These solutions are not the best (seeing description for every opcode is kinda distracting, you can have no access to the internet + googling takes more time).

**Describe the solution you'd like**
To add "Show description" action in context menu.

**Comment by ITAYC0HEN** [comment]
Good idea, and shouldn't be complicated to implement.
"Show opcode description"

**Comment by karliss** [comment]
Context menu is quite crowded already. Here are some alternative ideas to consider.

### Making it easier to toggle `asm describe`. Submenu under main view menu.
   +no extra UI for displaying description
   +no extra code for querying description string from r2
   -still quite slow as it requires at least two clicks for enabling and two for disabling asm.describe
### Tooltip when hovering instruction
   +no extra steps for triggering
    -bad for long descriptions
    -not sure if cutter has sufficient information about which tokens are opcodes
### Some kind F1 based shortcut, 
F1 + subset of modifier keys. For discoverability purpose probably should be also put under the help menu.
    + easy to trigger
    - Managing the state of action under global menu might be a bit messy.

**Comment by Maijin** [comment]
Yeah i also think adding to context menu is a bad idea because it's already very crowded there.

**Comment by karliss** [comment]
One more  idea for showing it
### Instruction description widget
Widget which show description for instruction at current seek offset
+ After opening the widget easy to see description for multiple instructions by clicking on them

**Comment by optizone** [comment]
I believe that we already have one, but I can't find it

**Comment by winadam** [comment]
I have an similar issue so let me know if I should make a separate issue.
I'm working on a project in PowerPC and using the asm.psuedo option but the opcodes (lbz, lbzu), (stw, and stwu) have the same descriptions but the u verison modifies a register. Could there the ability to see the actual opcode when I hover on the psuedo code version?

**Comment by Maijin** [comment]
That's unrelated to this issue, please open a new one.

**Comment by ITAYC0HEN** [comment]
> I believe that we already have one, but I can't find it
Someone wrote a plugin for it
https://github.com/daringjoker/Assembly-refrence

**Comment by xarkes** [comment]
So, should we close this?

**Comment by ITAYC0HEN** [comment]
@karliss suggested many good ways, we need to pick one and go for it. I actually like how the assembly-refrence plugin looks like, it is not based on asm.describe though. This is basically the 4th suggestion of @karliss

![image](https://user-images.githubusercontent.com/20182642/65672372-93e8ef80-e051-11e9-95d1-bcdba4795c6d.png)

**Comment by Maijin** [comment]
IMHO, this should be implemented on r2 side and exposed in cutter.

The asm.describe databases are very easy for new comers to contribute to and that's why there are so many architecture supported.

**Comment by ITAYC0HEN** [comment]
> this should be implemented on r2 side and exposed in cutter like suggested

In what shape? What changes should take place in r2 in addition to the current asm.descirbe?

**Comment by Maijin** [comment]
change in the databases to have more granularity + change in the display of e asm.describe and `?d`

**Comment by ITAYC0HEN** [comment]
Cutterref by @yossizap is also neat

https://github.com/yossizap/cutterref

> At the moment it only supports x86-64, ARM and MIPS 32bit

![image](https://user-images.githubusercontent.com/20182642/73814201-08dfe900-47eb-11ea-8bcc-bd0d019393d1.png)

**Comment by ITAYC0HEN** [comment]
Should we do a survey for the best option? or keep it to plugins?

**Comment by Maijin** [comment]
IMHO current solution is enough, the changes should be done in radare2 side to integrate more details in `?d` in future but for now having this cutteref offers enough for a temporary fix.

**Comment by karliss** [comment]
Current list of plugins can be found here https://github.com/rizinorg/cutter-plugins/?tab=readme-ov-file#enhancements

**Comment by axelkar** [comment]
If someone implements this for hover, I recommend taking inspiration from IDEs' signature hover functionality. You can scroll documentation in the little popup and it doesn't go away when you move your cursor on top of it.
